frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/code_block/code_block_shortcut_event.dart

```diff
@@ -264,3 +278,30 @@ CommandShortcutEventHandler _selectAllInCodeBlockCommandHandler =
 
   return KeyEventResult.handled;
 };
+
+CommandShortcutEventHandler _pasteInCodeBlock = (editorState) {
+  final selection = editorState.selection;
+  if (selection == null || !selection.isCollapsed) {
```

Comment:
if selection isn't collapsed, we replace the selection with the pasted content, we shouldn't ignore the action, right?